app/static/css/header.css

```
@@ -106,3 +106,95 @@ header {
     width: 100%;
     flex: 1
 }
+
+.desktop-tap-div {
+    border-radius: 0 0 8px 8px;
+    box-shadow: 0 2px 3px rgba(32, 33, 36, 0.18);
+    margin-bottom: 10px;
+    overflow: hidden;
+}
+
+.bRsWnc {
+    background-color: #fff;
```

Is this being applied? It's likely getting overwritten by the Whoogle theme. 

```suggestion
    background-color: #fff;
```

---

Maybe not, I have just taken all the css related to the tabs, I need to do some cleaning.